src/main/resources/resources/logisim/strings/std/std_de.properties

```diff
@@ -291,29 +291,28 @@ ioShapeSquare = Quadratisch
 ioShapePaddedSquare = Umrandetes Quadrat
 ioLedBarComponent = LED Leiste
 ioLedBarSegments = Segmente
-# ==> ioLedBarInput =
-# ==> ioLedBarInputSeparated =
-# ==> ioLedBarInputOneWire =
+ioLedBarInput = Eingangs Form:
+ioLedBarInputSeparated = Getrennt
+ioLedBarInputOneWire = Eine leitung
 #
 # io/extra/Buzzer.java
 #
 buzzerFrequecy = Frequenz
 buzzerVolume = Volumen
 buzzerVolumeBitWidth = Volumenbitbreite
 buzzerUnitDhz = dHz (0,1Hz)
-# ==> buzzerWaveform =
-# ==> buzzerSine =
-# ==> buzzerSquare =
-# ==> buzzerSmoothLevel =
-# ==> buzzerSmoothWidth =
-# ==> buzzerChannel =
-# ==> buzzerChannelBoth =
-# ==> buzzerChannelLeft =
-# ==> buzzerChannelRight =
-# ==> buzzerTriangle =
-# ==> buzzerSawtooth =
-# ==> buzzerNoise =
-# ==> buzzerDutyCycle =
+buzzerWaveform = Signalform
+buzzerSine = Sinus
+buzzerSquare = Rechteck
+# ==> buzzerSmoothLevel = Lautstärke Glätten
```

Comment:
I'm not sure what those two options make.
Can someone explain and maybe give me the correct translation.

Reply:
Which exact two out four you posted you mean?

Reply:
the two outcommented ones.
`buzzerSmoothLevel`
`buzzerSmoothWidth`

Reply:
Sure, these are related to the buzzer component, which is a simple waveform generator.

It supports sines, squares, triangles, sawteeth and noise.

It also supports a moving average filter for smoothing the signal. That filter has a degree (number of iterations) and a width (size of the moving window)

Reply:
I would translate them with "Glättungs stärke" and "Fenster größe".
@maehne can you think of better ones?